test/xkeyboard-config-test.py.in

```diff
@@ -12,12 +12,16 @@ verbose = True
 DEFAULT_RULES_XML = '@XKB_CONFIG_ROOT@/rules/evdev.xml'
 
 # Meson needs to fill this in so we can call the tool in the buildir.
-EXTRA_PATH='@MESON_BUILD_ROOT@'
+EXTRA_PATH = '@MESON_BUILD_ROOT@'
 os.environ['PATH'] = ':'.join([EXTRA_PATH, os.getenv('PATH')])
 
 
 # The function generating the progress bar (if any).
```

Comment:
This comment should be above `progress_bar`

Reply:
done